msteams-platform/m365-apps/publish.md

```diff
@@ -1,18 +1,18 @@
 ---
 title: Publish Teams apps for Microsoft 365
-description: Learn how to make your Microsoft 365-enabled Teams apps discoverable to users in Teams, Outlook, and Office through single tenant and multitenant distribution.
+description: Learn how to make your Microsoft 365-enabled Teams apps discoverable to users in Teams, Outlook, and Microsoft 365 through single tenant and multitenant distribution.
 ms.date: 10/10/2022
 ms.topic: conceptual
 ms.custom: m365apps
 ms.localizationpriority: medium
 ---
 # Publish Teams apps for Microsoft 365
 
-Microsoft Teams supports Microsoft 365-enabled Teams apps for production. You can distribute these apps to users who use the *Targeted Release* (dev preview) versions of Outlook.com and Office.com, the *Beta Channel* build of Outlook for Windows desktop, and Office Current Channel (dev preview) build of Office app for Android. Distribution options and processes for Microsoft 365-enabled Teams apps are the same as for traditional Teams apps.
+Microsoft Teams supports Microsoft 365-enabled Teams apps for production. You can distribute these apps to users who use the *Targeted Release* (dev preview) versions of Outlook.com and microsoft365.com, the *Beta Channel* build of Outlook for Windows desktop, and Microsoft 365 Current Channel (dev preview) build of Microsoft 365 app for Android. Distribution options and processes for Microsoft 365-enabled Teams apps are the same as for traditional Teams apps.
```

Comment:
Let's mention the former domain name too:

versions of outlook.com and microsoft365.com (formerly office.com),